### DISCUSSION TITLE
安卓，使用官方提供的PaddleOCR项目报错

### DISCUSSION THREAD

**Discussion by gaotianzeng** [PaddlePaddle/PaddleOCR#12313] [Q&A]
paddleLite版本：v2.13-rc
模型：ch_PP-OCRv4_det、ch_PP-OCRv4_rec

将模型替换为自己训练的.nb文件并将项目中的paddleLite替换为2.13，打开app后就报错，请问是什么原因？

**Comment by wxk1234567** [comment]
你好，你在用官方Android的demo可以打开吗？我会出现闪退

**Comment by saikewei** [comment]
我的也是请问你解决了吗，我是把原来的2.10rc替换为2.12版本就会报错，loadlibrary之后就会Fatal signal 11 (SIGSEGV), code 1, fault addr 0x0 in tid 1145 (hineseinterpret)
pid: 1145, tid: 1145, name: hineseinterpret  >>> com.example.chineseinterpret <<<
    #01 pc 0000000000094d2c  /data/app/com.example.chineseinterpret-wD1uDJQ6Pc4dgN3SXRdpug==/base.apk (offset 0x61f91000)
activityResumeTrigger: not whiteListedcom.example.chineseinterpret/com.example.chineseinterpret.LoginActivity/1
No package identifier when getting name for resource number 0x00000000
